projects/laji/src/app/shared-modules/document-viewer/document/document.component.ts

```diff
@@ -198,6 +198,7 @@ export class DocumentComponent implements AfterViewInit, OnChanges, OnInit, OnDe
       .subscribe(({doc, rights}) => {
           this.hasEditRights = rights.hasEditRights;
           this.hasDeleteRights = rights.hasDeleteRights;
+        console.log(rights);
```

Comment:
This should be removed

Reply:
Thanks, removed also another console.log elsewhere in the same commit